img/common.js

```diff
@@ -97,6 +97,14 @@ function ipchoosewin(hostparams) {
 	return openSelectWindow2(url, 'chooseip', 350, 380, 'true', ipelem, netelem);
 }
 
+function long2ip(ip) {
+    if (!isFinite(ip)) {
+        return false;
+    }
+
+    return [ip >>> 24, ip >>> 16 & 0xFF, ip >>> 8 & 0xFF, ip & 0xFF].join('.');
```

Comment:
To nie zadziała z PHP 5.3.
Wszędzie zamiast join/split używamy implode/explode.